src/main/java/sysc4806/project24/mini_survey_monkey/models/Survey.java

```diff
@@ -0,0 +1,57 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+public class Survey {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int surveyId;
```

Comment:
**Could we change this from `surveyId` to `id`?**

Argument for change would be to eliminate redundant words.